charts/portworx/templates/portworx-rbac-config.yaml

```diff
@@ -23,13 +23,22 @@ rules:
     verbs: ["delete", "get", "list", "watch", "update"]
   - apiGroups: [""]
     resources: ["persistentvolumeclaims", "persistentvolumes"]
-    verbs: ["get", "list"]
+    verbs: ["get", "list", "create", "delete", "update"]
   - apiGroups: ["storage.k8s.io"]
-    resources: ["storageclasses"]
+    resources: ["storageclasses", "csinodes"]
     verbs: ["get", "list"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["volumeattachments"]
+    verbs: ["get", "list", "create", "delete", "update"]
   - apiGroups: [""]
     resources: ["configmaps"]
     verbs: ["get", "list", "update", "create"]
+  - apiGroups: [""]
+    resources: ["services"]
+    verbs: ["get", "list", "create", "update", "delete"]
+  - apiGroups: [""]
+    resources: ["endpoints"]
+    verbs: ["get", "list", "create", "update", "delete"]
```

Comment:
nit: keep the order of verbs the same everywhere

nit: specify services and endpoints in the same section like pvc and pv

Reply:
this is the copied spec from px-installer. I think we should keep the same consistency between specs even if it's not best practices.